modules/picarto.js

```diff
@@ -159,6 +212,7 @@ function picarto_connection(_token, _debug) {
 	};
 }
 
+
```

Comment:
Mind removing this newline? It's just cosmetics, but something that you could do while you are editing the file anyways, if you don't mind.